include/net/buf.h

```diff
@@ -91,7 +91,7 @@ struct net_buf_simple {
 	/** Length of the data behind the data pointer. */
```

Comment:
typo in the commit msg, s/curser/cursor/
Also you can shorten the commit subject as the "context" is not really needed here, just write it
`net: pkt: Respect reserved bytes`

Reply:
fixed